File: 30 days of code JAVA/day13.java
// Declare your class here. Do not use the 'public' access modifier.
    // Declare the price instance variable
    
    /**   
    *   Class Constructor
    *   
    *   @param title The book's title.
    *   @param author The book's author.
    *   @param price The book's price.
    **/
    // Write your constructor here
    
class MyBook extends Book
{     int price;
    public MyBook(String title,String author,int pri)
    {
        super(title,author);
        price=pri;
    }
   
   void display()
   {
       System.out.println("Title: "+title);
       System.out.println("Author: "+author);
       System.out.println("Price: "+price);
       
   }
 
}

    /**   
    *   Method Name: display
    *   
    *   Print the title, author, and price in the specified format.
    **/
    // Write your method here
    
// End class

File: 30daysOFcode_JavaScript/day 13.js
class Book {
    constructor(title, author) {
        this.title = title;
        this.author = author;
    }
}

// Declare your class here.
class MyBook extends Book {
    /**
     *   Class Constructor
     *
     *   @param title The book's title.
     *   @param author The book's author.
     *   @param price The book's price.
     **/
    // Write your constructor here
    constructor(title, author, price) {
        super(title, author);
        this.price = price
    }

    /**
     *   Method Name: display
     *
     *   Print the title, author, and price in the specified format.
     **/
    // Write your method here
    display() {
        console.log('Title: ' + this.title + '\n' + 'Author: ' + this.author + '\n' + 'Price: ' + this.price);
    }

    // End class
}

module.exports = MyBook;
